mcp-server/README.md

```
@@ -0,0 +1,95 @@
+| command | `uvx` |
+| args | `["carabassa-mcp"]` |
+| env | `CARABASSA_BASE_URL`, `CARABASSA_WRITE_ENABLED` |
+
+> Before publishing to PyPI, use args `["--from", "/path/to/carabassa/mcp-server", "carabassa-mcp"]`.
+
+Set `CARABASSA_WRITE_ENABLED` to `"true"` to enable create/update/delete tools.
+
+## Available tools
+
+### Datasets
+| Tool | Description |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)